Sources/ConsoleKit/Console.swift

```diff
@@ -47,6 +47,21 @@ public protocol Console: class {
     ///
     /// - parameters:
     ///     - secure: If `true`, the input should not be shown while it is entered.
+    ///
+    /// - returns: the string that was read, or `nil` if EOF was encountered
+    func read(isSecure: Bool) -> String?
```

Comment:
Couldn't this be added as an extension that in the next major version will be added to the protocol?

Reply:
Ah, dumb question. I see that there's plenty of internals needed to support the method than can't be done in a protocol extension